### DISCUSSION TITLE
AAX Audio Converter Abort

### DISCUSSION THREAD

**Discussion by Original-Runsha** [audiamus/AaxAudioConverter#158] [Q&A]
Since Audible decided to do away with PC Support / downloads I have been using AaxAudioConverter with Book Lib Connect to try and save my library of 1000+ Audible purchases before they become totally unusable. (Live in mountains with zero cellular support so no smart phone) And I have celebrated these two products every day. Until last night. I suddenly have one title that consistently fails convert, even after being re-downloaded and retried. I attempted to run with logging, but for some reason there is never a log file created. So all I have are a few screenshots to hopefully help us figure out what is going on. 

First file is what happens when I run with just this one troublesome selection.
Second file is the produced output directory. It produces just the MP3 file (which appears to be complete but without any metadata) but not the jpeg cover image or the text file with the stripped ID tags and book info.
While I can, with a biy of effort, manually recover the image and metadata and updaye the tags on the audio file. I thought the issue should be reported in case mine is not a unique instance. 

![Abort Error](https://user-images.githubusercontent.com/119818852/205518556-1d5769b4-48d8-48d4-ae98-3b280189779a.jpg)
![Produced Output](https://user-images.githubusercontent.com/119818852/205518568-433d58c7-e6f6-4624-af6b-66856df78b9f.jpg)

**Comment by audiamus** [comment]
During the Book Lib Connect process, the book will be downloaded and decrypted and a plain .m4b file will be created. Afterwards, if export to AAX is enabled, Book Lib Connect will copy and rename the .m4b file to .aax and put additional meta data into separate .json files for AAX Audio Converter to digest. 

If something went wrong during that workflow you can try to backtrack. Find the .m4b file in the the Book Lib Connect download folder and try to play it. If that works, go to the Book Lib Connect export folder, find the .aax file and rename in back to .m4b. Try to play it. If that works as well, it could also be the meta data. We then really need the AAX Audi Converter log file for further investigation.
